Marlin/Conditionals_LCD.h

```diff
@@ -108,7 +108,26 @@
     #define ULTIPANEL
     #define NEWPANEL
 
-  #endif
+  #elif ENABLED(CR10_stockdisplay)
+  
+    #define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+    #define ST7920_DELAY_1 DELAY_2_NOP
+    #define ST7920_DELAY_2 DELAY_2_NOP
+    #define ST7920_DELAY_3 DELAY_2_NOP
```

Comment:
These must be:
```cpp
    #ifndef ST7920_DELAY_1
      #define ST7920_DELAY_1 DELAY_2_NOP
    #endif
    #ifndef ST7920_DELAY_2
      #define ST7920_DELAY_2 DELAY_2_NOP
    #endif
    #ifndef ST7920_DELAY_3
      #define ST7920_DELAY_3 DELAY_2_NOP
    #endif
```

Reply:
fiveangle had me remove them  Which is it?

Reply:
I am not following the thread well, apparently.  What note below?

Reply:
I thought Scott indicated that Conditionals_LCD.h was pre-loaded prior to Config*.h and pins_[board].h so didn't need these checks.  If he says they are needed, so be it.  Sorry for missing this earlier Jim.  Whatever Scott says should over-ride what others say (because even when he's wrong, he will be the final one to catch it ;)

Reply:
ok! :)